projects/dsp-ui/src/lib/action/components/message/message.component.html

```diff
@@ -0,0 +1,46 @@
+<mat-card *ngIf="!short" class="fix-width dsp-message" [ngClass]="'dsp-' + message?.type">
+
+    <mat-card-subtitle class="message-subtitle">
+        <span class="left">{{message?.type | uppercase }} {{message?.status}} | {{message?.statusMsg}}</span>
+        <!-- <mat-icon class="right">{{message?.type}}</mat-icon> -->
+    </mat-card-subtitle>
+
+    <mat-card-title class="message-title" [innerHtml]="message?.statusText"></mat-card-title>
+
+    <mat-card-content class="message-content">
+        <mat-list *ngIf="message?.route">
+            <mat-list-item>
+                <a [href]="message?.route" target="_blank">
+                    &rarr; {{message?.route}}
+                </a>
+            </mat-list-item>
+        </mat-list>
+
+        <mat-list *ngIf="showLinks">
+            <p>{{links.title}}</p>
+            <mat-list-item *ngFor="let item of links.list" class="link" (click)="goToLocation(item.route)">
+                <mat-icon mat-list-icon>{{item.icon}}</mat-icon>
+                <p mat-line>{{item.label}}</p>
+            </mat-list-item>
+        </mat-list>
+
+    </mat-card-content>
+
+    <mat-card-footer *ngIf="!medium" class="message-footnote" [innerHtml]="message?.footnote"></mat-card-footer>
+
+</mat-card>
+
+<mat-card *ngIf="short && !disable" class="fix-width dsp-short-message" [ngClass]="'dsp-' + message?.type"
+          (click)="closeMessage()">
+
+    <div class="dsp-panel">
+        <span class="dsp-short-message-text">
+            {{message?.statusText}}
+        </span>
+        <span class="fill-remaining-space"></span>
+        <!--         <button mat-icon-button (click)="closeMessage()">
```

Comment:
do we need to keep this commented code block??

Reply:
Most likely we don't but I wanted to keep everything as close to identical to the old code as possible. This was commented out in the old code as well. I would assume it's safe to remove?

Reply:
I suppose yes
@kilchenmann can we remove this code block?

Reply:
It was an icon `x` to close the message. But I see that a click event on the whole card is set to close the message. So, we don't need the icon anymore. You can delete this commented code block...